media/css/firefox/lockwise/lockwise.scss

```
@@ -6,52 +6,62 @@ $font-path: '/media/fonts';
 $image-path: '/media/protocol/img';
 
 @import '../../../protocol/css/includes/lib';
-@import '../../../protocol/css/components/hero';
 @import '../../../protocol/css/components/picto';
+@import '../../../protocol/css/components/split';
 @import '../../../protocol/css/templates/card-layout';
 @import '../../../protocol/css/templates/multi-column';
+@import '../../../protocol/css/components/logos/wordmark';
+@import '../../../protocol/css/components/logos/wordmark-product-lockwise';
 
+// split
 
-.add-on-download p {
-    margin: $spacing-sm;
+.mzp-c-split {
+    padding: $spacing-2xl;
 }
 
-.card-header {
-    text-align: center;
+.mzp-c-split-container {
+
+    p {
+        font-size: 18px;
```

Nit: try and use Protocol mixins for type sizing consistency?

```suggestion
        @include text-body-lg
```